web/themes/custom/liibarrio2020/liibarrio2020.libraries.yml

```diff
@@ -27,11 +27,17 @@ nicescroll:
 
 legislation-content:
   version: VERSION
-  css:
-    theme:
-      css/indigo-web.css: {}
   js:
-    js/legislation.js: {}
+    https://cdn.jsdelivr.net/gh/laws-africa/la-web-components@v0.3.0-beta/dist/la-web-components/la-web-components.esm.js: {
```

Comment:
This version must reference the bundle.js file so that pocketlaw does the right thing.

https://cdn.jsdelivr.net/gh/laws-africa/la-web-components@0.3.0-beta/dist/la-web-components/la-web-components.bundle.js

The nomodule version doesn't need to change.

Reply:
changed type module to reference the v0.3.0 la-web-components.bundle.js
but nomodule does not. just references the normal  v0.3.0 la-web-components.js